roles/basic_prog_pkgs/vars/main.yml

```
@@ -1,18 +1,14 @@
 ---
 # vars file for basic-prog-pkgs
 basic_prog_pkgs_intro_development:
- - artha
- - bless
- - chromium-browser
- - fonts-crosextra-caladea
- - fonts-crosextra-carlito
- - geany
- - idle
+ - flake8
  - libreoffice
- - libreoffice-templates
- - libreoffice-style-tango
  - logisim
  - meld
+ - python3-flake8-docstrings
```

```suggestion
 - python3-flake8-docstrings
 - python-is-python3
```

Maybe it's not ideal from a like correctness and portability perspective but it _does_ seem like it'd be more user friendly? Maybe we should even add this to `common` or `user`?

---

I don't want to get into installing tools outside a course's recommendations, but these seemed like helpful cleanups of what we're offering.